### ISSUE TITLE
Wrong loop keyword in LEARN.md, probably typo

### ISSUE THREAD

**Issue by mustiboost** [rabbibotton/clog#203]
Hi David,

In Chapter 8 on page 2 it says: 

"for N from 1 to 5 step 2"

There is no for keyword called "step". Did you mean to write: 

"for N from 1 to 5 by 2 ; step 2"  
? 

Also, it might be helpful to paginate the documents so we can refer to them by page number in the future when discussing issues or if someone wants to quote something from your documents.

**Comment by rabbibotton** [comment]
fixed. Thanks!

I plan on redoing at some point and will for sure try to use a better format, maybe as you say. or adding section and subsection number or the like.

Thanks!!!

**Comment by mustiboost** [comment]
No problem at all! I had noticed some other issues a few weeks/months ago when I browsed through your various documents, but failed to write it all down. I may go through your documents again specifically for the purpose of documenting such issues in the future. I'll let you know about it here of course. :)  

I love what you're doing, and want to get involved more in the future. I am actually in the process of wrapping webview.h at https://github.com/webview/webview, and I figured out how to make the lisp bindings work (and create executables) for windows and linux (mac is a different story...cocoa does not play nice with it not being in the main thread, so need to figure that out), but it's still in its early stages.  I specifically thought about wrapping webview.h so that we can have an easy way of having self-contained CLOG desktop executables that work independently from a browser...without having to resort to electron/ceramic. 

Love what you're doing, keep up the good work! :)
